opaque_keys/edx/django/models.py

```diff
@@ -121,8 +121,7 @@ def to_python(self, value):
                 ))
                 value = value.rstrip()
             return self.KEY_CLASS.from_string(value)
-        else:
-            return value
+        return value
```

Comment:
Question: Has there been an intentional pylint change to now require `no-else-return`?

Reply:
I think that was a side effect of upgrading pylint and/or pytest-pylint.  That check is relatively new, and edx-lint doesn't say anything about whether it should be run by default or not.